lib/iris/tests/results/analysis/first_quartile_foo_bar_2d.cml

```diff
@@ -8,13 +8,11 @@
       <coord>
         <dimCoord bounds="[[-15, 45]]" id="549be28b" long_name="foo" points="[15.0]" shape="(1,)" units="Unit('1')" value_type="float64"/>
       </coord>
+      <coord>
+        <dimCoord id="38918495" long_name="percentile_over_foo_bar" points="[25]" shape="(1,)" units="Unit('1')" value_type="int64"/>
+      </coord>
     </coords>
-    <cellMethods>
-      <cellMethod method="percentile (25%)">
-        <coord name="foo"/>
-        <coord name="bar"/>
-      </cellMethod>
-    </cellMethods>
-    <data dtype="float64" shape="()" state="loaded"/>
+    <cellMethods/>
+    <data dtype="float64" shape="(1,)" state="loaded"/>
```

Comment:
I'm pretty sure it should be scalar. If you run the following you'll see what I mean:

```python
c = iris.cube.Cube(np.arange(24))
c.add_dim_coord(iris.coords.DimCoord(range(24), 'time'), 0)

print c
print c.collapsed('time', iris.analysis.PERCENTILE, percent=50)
print c.collapsed('time', iris.analysis.MEAN)
```